Refresh.GameServer/Types/Activity/EventType.cs

```diff
@@ -0,0 +1,50 @@
+using System.Xml.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Refresh.GameServer.Types.Activity;
+
+[JsonConverter(typeof(StringEnumConverter))]
+public enum EventType
+{
+    [XmlEnum("publish_level")]
+    LevelUpload = 0,
+    [XmlEnum("heart_level")]
+    LevelFavourite = 1,
+    [XmlEnum("unheart_level")]
+    LevelUnfavourite = 2,
+    [XmlEnum("heart_user")]
+    UserFavourite = 3,
+    [XmlEnum("unheart_user")]
+    UserUnfavourite = 4,
+    [XmlEnum("play_level")]
+    LevelPlay = 5,
+    [XmlEnum("rate_level")]
+    LevelRate = 6,
+    [XmlEnum("tag_level")]
+    LevelTag = 7,
+    // [XmlEnum("comment_on_level")]
+    // PostLevelComment = 8,
+    // [XmlEnum("delete_level_comment")]
+    // DeleteLevelComment = 9,
+    // [XmlEnum("upload_photo")]
+    // PhotoUpload = 10,
+    // [XmlEnum("unpublish_level")]
+    // LevelUnpublish = 11,
+    // [XmlEnum("news_post")]
+    // NewsPost = 12,
+    [XmlEnum("mm_pick_level")]
+    LevelTeamPick = 13,
+    [XmlEnum("dpad_rate_level")]
+    LevelDpadRate = 14,
+    [XmlEnum("review_level")]
+    LevelReview = 15,
+    // [XmlEnum("comment_on_user")]
+    // PostUserComment = 16,
+    // [XmlEnum("create_playlist")]
+    // PlaylistCreate = 17,
+    // [XmlEnum("heart_playlist")]
+    // PlaylistFavourite = 18,
+    // [XmlEnum("add_level_to_playlist")]
+    // PlaylistAddLevel = 19,
```

Comment:
why are a bunch of these commented out?

Reply:
These are for when these events are implemented in the future - uncommenting some will cause compilation errors (the source generator tries to implement types that don't yet exist, for example PhotoUpload will cause GamePhoto to be used)

Others are simply irrelevant, for example LevelUnpublish. I'm not sure if I'll end up implementing it in the future or not.

It's also to make sure people know what ids are already being used/potentially being used incase extra events are added in the future.

So I think it's okay to leave these here.